lib/docker-sync/dependencies/fswatch.rb

```diff
@@ -0,0 +1,14 @@
+module DockerSync
+  module Dependencies
+    module Fswatch
+      def self.available?
+        return @available if defined? @available
```

Comment:
`fswatch` does exists on linux.. so we can't do `return false unless Environment.mac?`..